djot_parser/examples/todo-list.html

```diff
@@ -1,11 +1,30 @@
 <ul class="task-list">
-<li class="unchecked">
+<li>
+<input disabled="" type="checkbox"/>
 unchecked
+<ul>
+<li>
+1
 </li>
-<li class="checked">
+<li>
+2
+</li>
+<li>
+3
+</li>
+</ul>
+</li>
+<li>
+<input disabled="" type="checkbox" checked=""/>
 checked
 </li>
-<li class="unchecked">
+<li>
+<input disabled="" type="checkbox"/>
+Inline <em>formatting (<a href="https://google.com">link</a>)</em>
+</li>
+<li>
+<input disabled="" type="checkbox"/>
 finish
+
```

Comment:
I'll figure out how to output HTML without this extra newline tomorrow, which may be the last change necessary

Reply:
I took another look at this, but I'm at a loss for where the bug is. It is definitely from the end of the file/list, but the changes I've tried have had side-effects. Any advice on what might be wrong?